### PR TITLE
Add Rollbar

### PR DESCRIPTION
**Pull request by joshuatobin**
CC: @lexelby @reidmix

### CODE REVIEW

File: lib/umpire/web.rb [hunk 1]
```diff
@@ -189,8 +189,7 @@ def create_aggregator(aggregation_method)
     end
 
     error do
-      e = env["sinatra.error"]
-      log(at: "internal_error", "class" => e.class, message: e.message)
```
Comment (reidmix) [inline lib/umpire/web.rb]: I think you can leave the log too for splunkin'

Comment (joshuatobin) [inline lib/umpire/web.rb]: Good call. :/

**Comment by lexelby** [comment]
:+1:

**Comment by jmervine** [comment]
@joshuatobin Can this be released?

**Comment by joshuatobin** [comment]
@jmervine yeah its ready

**Comment by jmervine** [comment]
@joshuatobin Once released, the ROLLBAR_ACCESS_TOKEN needs to be set, correct? Not sure where to get that?

**Comment by joshuatobin** [comment]
Yup the token is the same, the DEPLOY env var needs to be different

**Pull request merged**